update examples readme (#62)
capeprivacy/pycape: examples/README.md

By default, Cape functions expect bytes as input and return bytes as output. For this first example, we run an echo function which expects bytes and return bytes as output.

To automatically deploy and run an example function that performs `echo`, run:
```
python3 deploy_run_echo.py
```

> Alternatively, you can `cape login` and `cape deploy` with Cape's CLI and set the following environment variables as follows. 

Reviewer: ```suggestion
Alternatively, you can use the Cape CLI directly via `cape login` and `cape deploy` as follows. 
```
